Detectors/TPC/reconstruction/macro/RawClusterFinder.C

```diff
@@ -0,0 +1,550 @@
+#ifndef ALICEO2_TPC_CONVERTRAWCLUSTERS_C_
```

Comment:
This is temporary code for the test beam Analysis. For this reason we didn't put it to the compiled code. At some point we might decide to remove it again. Cluster finding will be done by the HLT, but for the moment we needed something temporary.
Not sure if at some point it will make sense to put it to the compiled code.